pkgs/applications/science/electronics/fped/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchgit
+, flex, bison, fig2dev, imagemagick, netpbm, gtk2
+, pkgconfig
+}:
+
+with lib;
+stdenv.mkDerivation rec {
+  name = "fped-${version}";
+  version = "unstable-2017-05-11";
+
+  src = fetchgit {
+    url = "git://projects.qi-hardware.com/fped.git";
+    rev = "fa98e58157b6f68396d302c32421e882ac87f45b";
+    sha256 = "0xv364a00zwxhd9kg1z9sch5y0cxnrhk546asspyb9bh58sdzfy7";
+  };
+
+  LEX="flex";
+  RGBDEF="${netpbm}/share/netpbm/misc/rgb.txt";
```

Comment:
Perhaps a comment declaring its necessity?

Reply:
I can't remember why it was necessary, so I've removed it and things still build...

Reply:
@expipiplus1 This is needed https://github.com/NixOS/nixpkgs/pull/51285#issuecomment-482683277.
Things still built because it was on a different linux distro and without sandboxing.

Corrected in https://github.com/NixOS/nixpkgs/pull/51285/commits/d571ec0df025dfcc17196065e08c157b3b284ee9